src/tasks/add-solution-component/add-solution-component-v0/task.json

```diff
@@ -50,6 +50,14 @@
       "visibleRule": "authenticationType = PowerPlatformSPN",
       "helpMarkDown": "Authenticate with your Power Platform environment with an Azure AppID, tenantID and client secret."
     },
+    {
+      "name": "Cloud",
+      "label": "Cloud",
+      "type": "string",
+      "required": false,
+      "groupName": "advanced",
+      "helpMarkDown": "Cloud instance to authenticate with. Default: Public. See 'pac auth create help' for valid cloud instance names"
```

Comment:
While `See 'pac auth create help' for valid cloud instance names` is useful in that it lists the Clouds PPBT's underlying PAC uses, a couple downsides of instructing users with that here:

1.  Not all users of the PPBT will necessarily have PAC already installed locally.  While I expect _most_ would, that's not a guarantee and could be confusing to a groups DRI (or equivalent) trying to debug a pipeline failure.
2. The version of PAC may drift, so a user's installed copy might not list the same as PPBT's copy of PAC does.

It would probably be more useful to have the list built-in to the help text here, though of course that has the other issue of needing to remember to update the list when PAC gets new options.

Reply:
So, I thought about that and built that out but then I noticed GitHub Actions has this formulation. If we can agree that a pickList is better, I can add that here and PP-A as well to bring them both into parity.